### PR TITLE
chore: adds temporary action to run tests on dev-{s,m,l}

### PR DESCRIPTION
**Pull request by nathanielc**

### CODE REVIEW

File: .github/workflows/run-durable-tmp.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    strategy:
+      matrix:
+        networks:
+          - keramik-ceramic-v4-dev-s
+          - keramik-ceramic-v4-dev-m
+          - keramik-ceramic-v4-dev-l
+    steps:
+      -
+        # We're piggybacking on this step to inject the CD manager job ID into this workflow
+        name: ${{ github.event.inputs.job_id }}
```
Comment (smrz2001) [inline .github/workflows/run-durable-tmp.yml:27]: Can remove these two lines.
```suggestion
```

File: .github/workflows/run-durable-tmp.yml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+          if [[ -n "$COMPOSEDB_URLS" ]];
+          then
+            sed -i "s|COMPOSEDB_URLS.*|&$COMPOSEDB_URLS|" suite/env/.env.dev
+            sed -i "s|CERAMIC_URLS.*|&$CERAMIC_URLS|" suite/env/.env.dev
+          fi
+          make DURABLE_ENV=dev durable-tests
+
+  collect-results:
+    name: Durable Test Results
+    if: ${{ github.event.inputs.job_id != null }}
```
Comment (smrz2001) [inline .github/workflows/run-durable-tmp.yml:68]: Should delete this line.
```suggestion
```

**Review by smrz2001 — APPROVED** [review]
Looks great! :shipit: 

Since it's a new workflow, you won't be able to test it before merging to `main` 😭 Looks good to me but in case something needs to be fixed, it'll have to be in a subsequent PR.

File: .github/workflows/run-durable-tmp.yml [hunk 3]
```diff
@@ -0,0 +1,78 @@
+name: Run Durable tests
+
+on:
+  schedule:
+    - cron: "0 0/8 * * *" # Every 8 hours
```
[the remark this answers is not in the document]
Comment (nathanielc) [reply .github/workflows/run-durable-tmp.yml]: Good call, done

**Review by smrz2001 — APPROVED** [review]
Looks great! :shipit:

Since it's a new workflow, you won't be able to test it before merging to main 😭 Looks good to me but in case something needs to be fixed, it'll have to be in a subsequent PR.

**Pull request merged**